### PR TITLE
Style organisation - gruntfile path - live style

### PR DESCRIPTION
**Pull request by chloeavoustin**

### CODE REVIEW

File: tests/MainBundle/Controller/DefaultControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/');
+
+        $this->assertContains('Hello World', $client->getResponse()->getContent());
```
Comment (akiavara) [inline tests/MainBundle/Controller/DefaultControllerTest.php]: You should change this line if you want to pass Travis tests (it check something that no longer exist)

File: src/CoreBundle/Resources/styles/sass/main.sass [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@import 'Commun/reset'
+@import 'Commun/variables'
+@import 'Commun/fontIcon'
+@import 'Commun/base'
+@import 'Commun/fixed'
```
Comment (ajardin) [inline src/CoreBundle/Resources/styles/sass/main.sass]: Commun => Common

File: src/CoreBundle/Resources/views/Live/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                            <div>
+                                <span class="video-timer"><span class="mongobox-timer"></span>3:46</span>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                {#Playlist container#}
+                {% block playlist %}
+                    {{ include ("CoreBundle:live:playlist.html.twig") }}
```
Comment (aureliengiry) [inline src/CoreBundle/Resources/views/Live/index.html.twig]: Remplace `CoreBundle:live:playlist.html.twig` par `CoreBundle:Live:playlist.html.twig`

File: src/CoreBundle/Resources/views/Live/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                {% endblock %}
+            </section>
+        {% endblock %}
+
+        {#Chat Preview - right part#}
+        {% block chatPreview %}
+            {{ include ("CoreBundle:chat:preview.html.twig") }}
```
Comment (aureliengiry) [inline src/CoreBundle/Resources/views/Live/index.html.twig]: Remplace `CoreBundle:chat:preview.html.twig` par `CoreBundle:Chat:preview.html.twig`

**Pull request merged**